tests/lib/rules/no-invalid-regexp.js

```diff
@@ -26,11 +26,11 @@ ruleTester.run("no-invalid-regexp", rule, {
         { code: "new RegExp('.', 'y')", options: [{ allowConstructorFlags: ["y"] }]},
         { code: "new RegExp('.', 'u')", options: [{ allowConstructorFlags: ["U"] }]},
         { code: "new RegExp('.', 'yu')", options: [{ allowConstructorFlags: ["y", "u"] }]},
-        { code: "new RegExp('\/', 'yu')", options: [{ allowConstructorFlags: ["y", "u"] }]},
+        { code: "new RegExp('\/', 'yu')", options: [{ allowConstructorFlags: ["y", "u"] }]}, // eslint-disable-line no-useless-escape
```

Comment:
I think we need to figure out the intent of this test. Either the regex is `/\\\/` (matching backslash followed by slash), in which case this string should say `'\\/'`; or the regex is `/\//` (matching slash), in which case the string should say `'/'`.